### PR TITLE
Avoid horizontal scroll on SettingsPageContentLayout component fixes #548

### LINKED ISSUE #548: Scroll in horizontal and verical bars when lauching mobilization

### PR DESCRIPTION
**Pull request by lpirola**

### CODE REVIEW

File: client/mobilizations/widgets/__plugins__/pressure/action-creators/async-fill-widget.js [hunk 1]
```diff
@@ -18,11 +18,12 @@ const asyncFillWidget = ({ payload: fill, widget }) => (dispatch, getState, { ap
 
   dispatch({ type: t.WIDGET_PRESSURE_FILL_REQUEST })
   return api.post(endpoint, body)
-    .then(response => {
+    .then(({ data }) => {
       dispatch({ type: t.WIDGET_PRESSURE_FILL_SUCCESS })
```
Comment (gabrielrtakeda) [inline client/mobilizations/widgets/__plugins__/pressure/action-creators/async-fill-widget.js]: I forgot to remove this line D:

File: client/mobilizations/widgets/__plugins__/pressure/reducers.js [hunk 2]
```diff
@@ -13,7 +19,15 @@ export default function PressureReducers (state = initialState, action = initial
     case t.WIDGET_PRESSURE_FILL_REQUEST:
       return { ...state, saving: true }
     case t.WIDGET_PRESSURE_FILL_SUCCESS:
-      return { ...state, saving: false, filled: true }
+      console.log('[client/mobilizations/widgets/__plugins__/pressure/reducers.js] action.payload', action.payload)
```
Comment (gabrielrtakeda) [inline client/mobilizations/widgets/__plugins__/pressure/reducers.js]: Another line that I forgot to remove :(

File: client/mobilizations/widgets/__plugins__/pressure/action-creators/async-fill-widget.js (hunk 1 above)
Comment (lpirola) [reply client/mobilizations/widgets/__plugins__/pressure/action-creators/async-fill-widget.js]: It's time to commit!

**Review by gabrielrtakeda — APPROVED** [review]
Let's deploy!

**Pull request merged**